modules/helfi_address_search/src/Plugin/views/area/AddressSearchInfo.php

```diff
@@ -91,7 +91,7 @@ public function render($empty = FALSE): array {
       $format = $this->options['succeed']['format'] ?? filter_default_format();
     }
     else {
-      $classes[] = 'unit-search__address-not-found';
+      $classes[] = 'unit-search__address-not-found hds-notification hds-notification--alert';
```

Comment:
I think the more correct way is to add each CSS class as a separate array element. (Although the end result is the same when CSS classes are printed.)
```
$classes[] = 'unit-search__address-not-found';
$classes[] = 'hds-notification';
$classes[] = 'hds-notification--alert';
```